src/stylesheets/blocks/_problem.scss

```diff
@@ -1,4 +1,94 @@
 .problem-section {
   background-color: color(mint-green);
+
+  @media (screen(lg)) {
```

Comment:
Might want to throw screen in your media queries `@media screen and (screen(lg))`. Only really important if you're concerned about print styles, but it doesn't hurt.

Reply:
This might be something I have to save for future reference since there are a log of media queries happening in these stylesheets, but what is the effect of not including `screen and` ?